lib/ReviewAppClient.js

```diff
@@ -36,11 +40,14 @@ class ReviewAppClient {
   async scale(app, formation) {
     const scalingoClient = await this.getClient();
     console.debug(`Scaling app ${app.name} to ${formation[0].amount} container(s)…`);
+
     try {
-      await scalingoClient.Containers.scale(app.name, formation);
+      const { operation } = await scalingoClient.Containers.scale(app.name, formation);
+
+      await this.pollOperationStatus({ operation, interval, maxAttempts });
       console.log(`App ${app.name} scaled successfully`);
     } catch (err) {
-      if (err._data && err._data.error === 'no change in containers formation') {
+      if ((err._data && err._data.error === 'no change in containers formation') || errr === 'no change in containers formation') {
```

Comment:
Y'a pas un "r" de trop dans `errr === 'no change in containers formation')` ?

Reply:
Tout à fait, merci !